content/python/concepts/scope/scope.md

```diff
@@ -0,0 +1,58 @@
+---
+Title: 'Scope'
+Description: 'The variable is only usable inside the region it is created is called as scope in python'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+  - 'Web Development'
+Tags:
+  - 'Lists'
+  - 'Arrays'
+  - 'Objects'
+  - 'Data Structures'
+  - 'Data Types'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+The variable is only usable inside the region it is created is called as **scope** in python. Now this region can be anything a class, method(function), loops, if statements, etc.
+
+## Types of Scopes
+ **Local Scope**: 
+ 
+ Suppose there is a function and a variable is initialized in it. Now this variable can only be used within that funtion not from outside the function.
+
+ ~~~
```

Comment:
Change all of the `~~~` to three back-ticks ` ``` `. That's how we format the code snippets for the live Docs site.

Reply:
Hii @Dusch4593 , i have committed all the suggestions that you have made so far
thank you for your review and suggesting me the changes.